package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "angular-http-auth",
-  "version": "1.2.3",
+  "version": "1.3.0",
```

Comment:
Hi @simison 
How do you think it should be done the `npm` way? Should I keep `1.2.3` version at `master` until new release and then upgrade to `1.3.0` or the Maven way to set a `-SNAPSHOT` between releases? Or something else?